src/Microsoft.Framework.HashCodeCombiner.Sources/project.json

```diff
@@ -0,0 +1,19 @@
+﻿{
+    "version": "1.0.0-*", 
+    "shared": "*.cs", 
+    "dependencies": { }, 
+    "frameworks" : {
+        "net45": { }, 
+        "dnx451": { }, 
+        "dnxcore50": {
+            "dependencies": {
+                "System.Runtime": "4.0.20-*"
+            } 
+        },
+        "dotnet": {
```

Comment:
We should address all the common projects and their unneeded listing of every framework known to man :smile: